utils/CssParser.js

```diff
@@ -0,0 +1,99 @@
+const FileHelper = require('../helpers/FileHelper');
+const stringInject = require('stringinject');
+
+class CssParser {
+  constructor() {}
+
+  exportToDirectory(directory, tokens) {
+    const self = this;
+
+    Object.entries(tokens).map((variableName) => {
+      const cssFilename = variableName[0] + '.css';
+  
+      let stream = new FileHelper().streamToFileDirectory(directory, cssFilename);
+  
+      stream.once('open', function (fd) {
+        stream.write(':root {' + '\n');
+  
+        if (variableName.keys.length > 0) {
+          const variableFileCSS = self.parse(variableName);
+          variableFileCSS.map((p) => {
+            p.map((m) => {
```

Comment:
Chose a better variable name, not just `p`

Reply:
Hi @osmarpetry ,

I've fixed the CSS parser, sorry I missed that one.